test/chainBridge/voteDepositProposal.js

```diff
@@ -14,9 +14,9 @@ const ERC20HandlerContract = artifacts.require("ERC20Handler");
 contract('Bridge - [voteDepositProposal with relayerThreshold > 1]', async (accounts) => {
     const originChainID = 1;
     const destinationChainID = 1;
-    const originChainRelayerAddress = accounts[1];
-    const originChainRelayer2Address = accounts[4];
-    const originChainRelayer3Address = accounts[5];
+    const originChainRelayerAddress = accounts[0];
+    const originChainRelayer2Address = accounts[3];
+    const originChainRelayer3Address = accounts[4];
     const depositerAddress = accounts[2];
     const destinationChainRecipientAddress = accounts[3];
```

Comment:
I don't think it matters but now `destinationChainRecipientAddress` has the same address as `originChainRelayer2Address`, did you need to update the assigned addresses for Geth to work?